lib/plug/debugger.ex

```diff
@@ -187,6 +187,7 @@ defmodule Plug.Debugger do
     session = maybe_fetch_session(conn)
     params = maybe_fetch_query_params(conn)
     {title, message} = info(kind, reason)
+    [headline | details] = String.split(message, "\n\n")
```

Comment:
I think we no longer need to split between headline and details, no?

Reply:
Rather just always format everything as code. :)

Reply:
I think so, done 👍🏻